spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -1650,6 +1650,16 @@ class CometExecSuite extends CometTestBase {
           spark.catalog.cacheTable("abc")
           val df = spark.sql("SELECT * FROM abc").groupBy("key").count()
           checkSparkAnswerAndOperator(df, includeClasses = Seq(classOf[CometSparkToColumnarExec]))
+          df.collect()
```

Comment:
nit: it would be good to add a comment explaining why we need this call to collect